src/phases/scale/MixPhases.yml

```diff
@@ -1,7 +1,13 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/product-perf"
 Description: |
-  TODO: TIG-3318
+  Phase defintions for the MixedWorkloadsGenny, which is a port of the mixed_workloads in the
+  workloads repo. https://github.com/10gen/workloads/blob/master/workloads/mix.js. It runs 4 sets of
+  operations, each with dedicated actors/threads. The 4 operations are insert, findOne, updateOne,
+  and deleteOne. Since each type of operation runs in a dedicated thread it enables interesting
+  behavior, such as reads getting faster because of a write regression, or reads being starved by
+  writes. The origin of the test was as a reproduction for BF-2385 in which reads were starved out
+  by writes.
```

Comment:
nit: "The 4 operations are insert, ...." --> "The 4 operations are `insertOne,` ...."

And add:
```
Keywords:
- Scale
- insert
- Find
- Update
- Delete
```